lblod/frontend-embeddable-notule-editor
# Add test for dynamically growing editors to test-app workspace

## Overview
This PR moves the test for dynamically growing editor instances out of the `public` folder into the `test-app` workspace.

### How to test/reproduce
- `npm start`
- Open the `multiple-growing-editors` page
- Check if the editors grow correctly in a dynamic way.

### Challenges/uncertainties
- I did not yet add all the styles to the new `test-app` page. This did not seem very urgent at the moment, we can still look later into integrated css files with the `webpack-html-plugin`



### Checks PR readiness
